src/extensions/sticky-header/bootstrap-table-sticky-header.js

```diff
@@ -107,9 +107,16 @@ $.BootstrapTable = class extends $.BootstrapTable {
     // show sticky when top anchor touches header, and when bottom anchor not exceeded
     if (top > start && top <= end) {
       // ensure clone and source column widths are the same
-      this.$stickyHeader.find('tr:eq(0)').find('th').each((index, el) => {
-        $(el).css('min-width', this.$header.find('tr:eq(0)').find('th').eq(index).css('width'))
-      })
+      // this.$stickyHeader.find('tr:eq(0)').find('th').each((index, el) => {
+      //   $(el).css('min-width', this.$header.find('tr:eq(0)').find('th').eq(index).css('width'))
+      // })
+      // "y" axis
+      this.$stickyHeader.find('tr').map(function(index, el) {
+        var axis_x = $(el).find('th');
+        axis_x.each(function(indexX, elX) {
+          $__default['default'](elX).css('min-width', _this4.$header.find('tr:eq('+index+')').find('th:eq('+ indexX+')').css('width'));
+        });
+      });
```

Comment:
No semicolons, please run the `pre-commit` script (e.g. if you use yarn `yarn pre-commit`) for linting etc.